### PR TITLE
Created CLI for config validation

### LINKED ISSUE #45: Create config file validation tool
It would be quite neat if we would be able to validate a review file without having to run the action itself.

### PR DESCRIPTION
**Pull request by Bullrich**
Created a very simple CLI which can validate a `review-bot.yml` config file and produce errors if it finds any.

Also added a step to evaluate the config file in the CI

This resolves #45

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,3 +5,12 @@
 [![Publish package to GitHub Packages](https://github.com/paritytech/review-bot/actions/workflows/publish.yml/badge.svg?branch=main)](https://github.com/paritytech/review-bot/actions/workflows/publish.yml)
 
 Have custom review rules for PRs with auto assignment.
+
+## Evaluating config
+
+If you want to evaluate the config, we have a simple `cli` to do so.
+
+```bash
+yarn run cli # this builds the cli
```
Comment (mutantcornholio) [inline README.md:14]: This sounds like it should execute cli, not like it builds it.

File: package.json [hunk 2]
```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "start": "node dist",
     "build": "ncc build --license LICENSE",
+    "cli": "ncc build src/cli.ts",
```
Comment (mutantcornholio) [inline package.json:9]: If we have more than one target, let's have more than one output.  
Action and cli shouldn't both be built at `dist`. 
I suggest naming them `review-bot` and `review-bot-cli`, respectively.

**Review by mutantcornholio — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (Bullrich) [reply package.json:9]: Could you elaborate on this? I'm not following.

What about changing it to this?
```suggestion
    "start": "node dist",
    "build": "ncc build --license LICENSE",
    "cli": "ncc build src/cli.ts && node dist",
```

My plan for this is to be a very minor QoL feature, don't want to release it.

Comment (mutantcornholio) [reply package.json]: > What about changing it to this?

same problem, both `cli` and action targets are built to `dist`. 
If you first do `yarn cli`, and then `yarn start`, you won't start the action. 

I suggest something like this: 

```json  
    "build": "ncc build --license LICENSE -o dist/review-bot",
    "start": "node dist/review-bot",
    "build:cli": "ncc build src/cli.ts -o dist/review-bot-cli",
    "start:cli": "node dist/review-bot-cli",
```

Comment (Bullrich) [reply package.json]: Ah makes sense!

I would like to keep the `dist` directory specifically for the review-bot.

Would `"cli": "ncc build src/cli.ts -o cli && node cli",` satisfy the requirement for you?

I would also add `/cli` to the `.gitignore`

Comment (mutantcornholio) [reply package.json]: Sure!
`dist-cli` is also an option =)

**Review by mutantcornholio — APPROVED** [review]

**Pull request merged**